controllers/statusreport/statusreport_controller_test.go

```diff
@@ -17,14 +17,14 @@ limitations under the License.
 package statusreport
 
 import (
-	"reflect"
-
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-appstudio/operator-toolkit/metadata"
```

Comment:
This looks like battle of IDEs, where one IDE is grouping it by type of import (stdlib, external) the other IDE just sort it alphabetically :|

Not a merge blocker

Reply:
Yeah, I'll look into IDE settings to see if we can sync our different setups in the future. Don't want to hold up this fix right now though.

Reply:
Resolving for now, will look into syncing the IDE settings within the team.